### ISSUE TITLE
test tab can't be rendered and throws error

### ISSUE THREAD

**Issue by boyangwang** [zalmoxisus/redux-devtools-test-generator#3]
I'm seeing the following errors when I click on "test" on redux-devtools-extension. This is happening for any page:

```
Uncaught TypeError: Cannot read property 'assertion' of undefinedvalue @ window.bundle.js:33_renderValidatedComponentWithoutOwnerOrContext @ window.bundle.js:6_renderValidatedComponent @ window.bundle.js:6performInitialMount @ window.bundle.js:6mountComponent @ window.bundle.js:6mountComponent @ window.bundle.js:5performInitialMount @ window.bundle.js:6mountComponent @ window.bundle.js:6mountComponent @ window.bundle.js:5_mountChildAtIndex @ window.bundle.js:6_updateChildren @ window.bundle.js:6updateChildren @ window.bundle.js:6_updateDOMChildren @ window.bundle.js:5updateComponent @ window.bundle.js:5receiveComponent @ window.bundle.js:5receiveComponent @ window.bundle.js:5_updateRenderedComponent @ window.bundle.js:6_performComponentUpdate @ window.bundle.js:6updateComponent @ window.bundle.js:6receiveComponent @ window.bundle.js:6receiveComponent @ window.bundle.js:5updateChildren @ window.bundle.js:6_reconcilerUpdateChildren @ window.bundle.js:6_updateChildren @ window.bundle.js:6updateChildren @ window.bundle.js:6_updateDOMChildren @ window.bundle.js:5updateComponent @ window.bundle.js:5receiveComponent @ window.bundle.js:5receiveComponent @ window.bundle.js:5_updateRenderedComponent @ window.bundle.js:6_performComponentUpdate @ window.bundle.js:6updateComponent @ window.bundle.js:6performUpdateIfNecessary @ window.bundle.js:6performUpdateIfNecessary @ window.bundle.js:5u @ window.bundle.js:5perform @ window.bundle.js:5perform @ window.bundle.js:5perform @ window.bundle.js:5S @ window.bundle.js:5closeAll @ window.bundle.js:5perform @ window.bundle.js:5batchedUpdates @ window.bundle.js:7i @ window.bundle.js:5dispatchEvent @ window.bundle.js:7
7window.html#right:1 Error in event handler for (unknown): TypeError: Cannot read property 'getHostNode' of null
    at Object.getHostNode (chrome-extension://lmhkpmbekcpmknklioeibfkpmmfibljd/js/window.bundle.js:5:6735)
    at getHostNode (chrome-extension://lmhkpmbekcpmknklioeibfkpmmfibljd/js/window.bundle.js:6:25630)
    at Object.getHostNode (chrome-extension://lmhkpmbekcpmknklioeibfkpmmfibljd/js/window.bundle.js:5:6736)
    at Object.updateChildren (chrome-extension://lmhkpmbekcpmknklioeibfkpmmfibljd/js/window.bundle.js:6:21501)
    at h._reconcilerUpdateChildren (chrome-extension://lmhkpmbekcpmknklioeibfkpmmfibljd/js/window.bundle.js:6:18759)
    at h._updateChildren (chrome-extension://lmhkpmbekcpmknklioeibfkpmmfibljd/js/window.bundle.js:6:19480)
    at h.updateChildren (chrome-extension://lmhkpmbekcpmknklioeibfkpmmfibljd/js/window.bundle.js:6:19383)
    at h._updateDOMChildren (chrome-extension://lmhkpmbekcpmknklioeibfkpmmfibljd/js/window.bundle.js:5:31739)
    at h.updateComponent (chrome-extension://lmhkpmbekcpmknklioeibfkpmmfibljd/js/window.bundle.js:5:30049)
    at h.receiveComponent (chrome-extension://lmhkpmbekcpmknklioeibfkpmmfibljd/js/window.bundle.js:5:29496)
```
subsequent calls will trigger

```
Uncaught TypeError: Cannot read property 'getHostNode' of nullgetHostNode @ window.bundle.js:5getHostNode @ window.bundle.js:6getHostNode @ window.bundle.js:5updateChildren @ window.bundle.js:6_reconcilerUpdateChildren @ window.bundle.js:6_updateChildren @ window.bundle.js:6updateChildren @ window.bundle.js:6_updateDOMChildren @ window.bundle.js:5updateComponent @ window.bundle.js:5receiveComponent @ window.bundle.js:5receiveComponent @ window.bundle.js:5_updateRenderedComponent @ window.bundle.js:6_performComponentUpdate @ window.bundle.js:6updateComponent @ window.bundle.js:6receiveComponent @ window.bundle.js:6receiveComponent @ window.bundle.js:5updateChildren @ window.bundle.js:6_reconcilerUpdateChildren @ window.bundle.js:6_updateChildren @ window.bundle.js:6updateChildren @ window.bundle.js:6_updateDOMChildren @ window.bundle.js:5updateComponent @ window.bundle.js:5receiveComponent @ window.bundle.js:5receiveComponent @ window.bundle.js:5_updateRenderedComponent @ window.bundle.js:6_performComponentUpdate @ window.bundle.js:6updateComponent @ window.bundle.js:6performUpdateIfNecessary @ window.bundle.js:6performUpdateIfNecessary @ window.bundle.js:5u @ window.bundle.js:5perform @ window.bundle.js:5perform @ window.bundle.js:5perform @ window.bundle.js:5S @ window.bundle.js:5closeAll @ window.bundle.js:5perform @ window.bundle.js:5batchedUpdates @ window.bundle.js:7i @ window.bundle.js:5dispatchEvent @ window.bundle.js:7
```

and test tab doesn't render. Extension version: 2.8.1.1

**Comment by zalmoxisus** [comment]
Just published `2.8.2`, where it should be fixed. Let me know whether it helps.

**Comment by boyangwang** [comment]
Fixed! Thanks
